Gaijinizer patch/CommonEvents_693.txt

```diff
@@ -3,329 +3,398 @@
+Hmm, I see thou hast become hard.
 
+Thine voice has also become shrill, it would seem
+this kind of action has made thee "aroused"...#Ron
 > END STRING
 
 > BEGIN STRING
 この図体のせいで、もっと細かく触ることは出来ぬが…。
 
 どれ……もう少し緩急を付けてみるか
-> CONTEXT: CommonEvents_693/82/Dialogue/enemy32 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/82/Dialogue/enemy32
+Given mine large frame, I cannot touch thee with
+any degree of finesse, but perhaps I can change
+the pace a little.#Ron
 > END STRING
 
 > BEGIN STRING
 ああああああっ…！！♡♡
 
 うああああああっ…！！♡
-> CONTEXT: CommonEvents_693/92/Dialogue/kazuya-0 < UNTRANSLATED
+> CONTEXT: CommonEvents_693/92/Dialogue/kazuya-0
+Aaaaaaah...!! ♡♡
 
+Uwaaaaaah...!! ♡#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]は、スフィンクスの指先でイッてしまった…
-> CONTEXT: CommonEvents_693/143/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/143/Dialogue
+Sphinx made \\N[1] cum with the tip of her finger...#Ron
 > END STRING
 
 > BEGIN STRING
 おっと……しまった。
 一気に触りすぎて出してしまったか…
 
 もう少しこの堅さを触って、調べてみたくもあったのだが…
-> CONTEXT: CommonEvents_693/145/Dialogue/enemy32 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/145/Dialogue/enemy32
+Oh dear, did I go too quickly all at once and
+make thee cum? I had wished to spend more time
+touching and investigating that hardness.#Ron
 > END STRING
 
 > BEGIN STRING
 しかし、これが人間の『精子』か……
 これで生殖行為を行うのだな……
 
 そして、我らが悪魔の一部には『食事』ともされている…
-> CONTEXT: CommonEvents_693/150/Dialogue/enemy32 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/150/Dialogue/enemy32
+So this is human "sperm"...
+This is what thou uses to reproduce, is it not?
+And part of what we demons turn into "food".#Ron
 > END STRING
 
 > BEGIN STRING
 はあっ…はあっ…♡
 
 お、お願い…スフィンクス…
 もう…やめ……
-> CONTEXT: CommonEvents_693/155/Dialogue/kazuya-0 < UNTRANSLATED
+> CONTEXT: CommonEvents_693/155/Dialogue/kazuya-0
+Haah... Haah... ♡
 
+P-Please... Sphinx...
+No... more...#Ron
 > END STRING
 
 > BEGIN STRING
 確か…『精子』は、短時間でも外気に晒されると
 すぐに死滅するのであったな……
 
 勿体ない……
-> CONTEXT: CommonEvents_693/160/Dialogue/enemy32 < UNTRANSLATED
+> CONTEXT: CommonEvents_693/160/Dialogue/enemy32
+If I recall correctly, "sperm" lasts only a brief
+time before dying when exposed to open air...
 
+Such a waste...#Ron
 > END STRING
 
 > BEGIN STRING
 ああっ…！！！
-> CONTEXT: CommonEvents_693/168/Dialogue/kazuya-0 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/168/Dialogue/kazuya-0
+Aaah...!!!#Ron
 > END STRING
 
 > BEGIN STRING
 スフィンクスは、僕の言うことに全く耳を貸さずに
 
 僕の身体を、ぐいっと持ち上げた……
-> CONTEXT: CommonEvents_693/170/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_693/170/Dialogue
+Not listening to a word I say, Sphinx suddenly lifts my
+whole body up...#Ron
 > END STRING
 
 > BEGIN STRING
 僕は、彼女の指先から抜け出すどころか
 股間をいじっている指ひとつ動かすことが出来ない…
 
 モフモフで、込めた力が吸収されてしまう…
-> CONTEXT: CommonEvents_693/174/Dialogue < UNTRANSLATED
+> CONTEXT: CommonEvents_693/174/Dialogue
+I can't even move the digit she has on my crotch, let
+alone slip out of her grasp...
 
+My strength has been sapped by the fluffiness...#RonIffy
 > END STRING
```

Comment:
![image](https://user-images.githubusercontent.com/48231102/112709539-18ebd980-8e80-11eb-8a18-4abf1bdae569.png)

This a valid interpretation of the second part?

Reply:
Yep, that works

Reply:
Okay then, let's formalize it:
```
I can't even move the digit she has on my crotch, let
alone slip out of her grasp...

It's so fluffy that it absorbs all the force I put into it...
```